client.go

```diff
@@ -143,13 +146,45 @@ func NewClient(dbDir string, cfg *ClientConfig) (*Client, func(), error) {
 		loopOutMaxParts:   cfg.LoopOutMaxParts,
 	})
 
+	// Create our manager config with functions which can query our current
+	// server terms.
+	mngrCfg := &liquidity.Config{
+		LoopOutRestrictions: func(ctx context.Context) (
+			*liquidity.Restrictions, error) {
+
+			outTerms, err := swapServerClient.GetLoopOutTerms(ctx)
+			if err != nil {
+				return nil, err
+			}
+
+			return liquidity.NewRestrictions(
+				outTerms.MinSwapAmount, outTerms.MaxSwapAmount,
+			), nil
+		},
+		LoopInRestrictions: func(ctx context.Context) (
+			*liquidity.Restrictions, error) {
+
+			inTerms, err := swapServerClient.GetLoopInTerms(ctx)
+			if err != nil {
+				return nil, err
+			}
+
+			return liquidity.NewRestrictions(
+				inTerms.MinSwapAmount, inTerms.MaxSwapAmount,
+			), nil
+		},
```

Comment:
Isn't it possible to just pass in `swapServerClient`?

Reply:
This needs a refactor at the moment because `swapServerClient` is in the loop package, so we'd have circular dependencies. Could also just move all of this into the loop package (?), or directly import `looprpc` (I don't think this is a good idea, since we have a mock for the interface already).